bin/get_data.py

```diff
@@ -50,8 +76,12 @@ def main(taxon_file, db):
             failed.append(index)
             os.rmdir(enddir)
     
+    print("\n   Building Ribosomic sequence database\n")
+    n = prepare_ribo_db(ribo_dir)
```

Comment:
new n overrides previous n which is not used at all

Reply:
Previous n is used to increase the progressbar which is completed by the time the genomes are downloaded, thus assigning a new value to n is does not affect the result. 

Want me to change it for the sake of readability tho?